cs3/storage/provider/v1beta1/resources.proto

```diff
@@ -64,8 +64,9 @@ message ResourceInfo {
   // For reference types this is NOT the mtime of the target.
   cs3.types.v1beta1.Timestamp mtime = 7;
   // REQUIRED.
-  // The path for the resource.
-  // It MUST start with the slash character (/).
+  // The path for the resource:
+  // MUST start with `/` when the reference had no resource_id, indicating an absolute path.
+  // MUST start with `.` when the reference had a resource_id, indicating a relative path.
```

Comment:
MUST be `` when reference is an ID and returned information is a full path.

Reply:
When you want to look up a path, the Reference used in the stat request MUST have a `reference_id` set and the `path` MUST be empty. But the `ResourceInfo` then MUST return the absolute path, which MUST start with `/`.

```
// MUST start with `.` when the reference had a resource_id and a path starting with `.`, indicating a relative path.
// Otherwise MUST start with `/` indicating an absolute path. 
```

I still don't like the naming of references: either they are absolute, or they are relative to the resource id. If we can agree on `relative reference` (id set & path starts with `.`) and `absolute reference` (id empty and path starts with `/`) I propose:

```
// MUST start with `/` when the call used an absolute reference (path starts with `/` & id not set).
// MUST start with `/` when the call used an relative reference (path starts with `.` & id set).
```

Reply:
> I still don't like the naming of references: either they are absolute, or they are relative to the resource id. If we can agree on relative reference (id set & path starts with .) and absolute reference (id empty and path starts with /)

Sounds good to me